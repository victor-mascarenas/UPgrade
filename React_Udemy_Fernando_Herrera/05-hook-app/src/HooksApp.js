import React from 'react';

const HooksApp = () => {
    return (
        <div>  
            <h1>Hola mundo</h1>
        </div>
    );
};

export default HooksApp;