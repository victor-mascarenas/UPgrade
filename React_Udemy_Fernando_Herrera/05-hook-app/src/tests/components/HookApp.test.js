import React from "react";
import {shallow} from 'enzyme';
import HooksApp from "../../HooksApp";

describe('Pruebas en HooksApp', () => {
    test('Debe empatar con el snapshot', () => {
        const wrapper = shallow(<HooksApp/>);
        expect(wrapper).toMatchSnapshot();
    });
    
});
