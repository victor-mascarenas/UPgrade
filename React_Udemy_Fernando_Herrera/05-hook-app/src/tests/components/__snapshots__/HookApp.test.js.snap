// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HooksApp Debe empatar con el snapshot 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
